tests/specs/benchmarks/verification.k

```diff
@@ -127,7 +127,8 @@ module VERIFICATION
   // Simplification
   // ########################
 
-    rule #asWord(#buf(32, DATA)) => DATA  requires #rangeUInt(256, DATA) [simplification]
+    // TODO: is generalized by `#asWord(#buf(N, BUF)) => BUF` in `lemmas.k`?
```

Comment:
no this doesn't work

Reply:
Hmmmmm, that's strange. Did it consistently fail on re-runs (and on your machine?)

The LLVM conformance test-suite doesn't even see this file.

Reply:
turns out I started a new CI job before this one finished, so that's why it failed